docs/_pages/releases.md

```diff
@@ -12,6 +12,7 @@ sidebar:
 **What's New**
 * Added `WithParameterName` extension to ease asserting on the parameter name for a thrown `ArgumentException` - [#1466](https://github.com/fluentassertions/fluentassertions/pull/1466).
 * Added `NotCompleteWithinAsync` to `TaskCompletionSourceAssertions` - [#1474](https://github.com/fluentassertions/fluentassertions/pull/1474).
+* Added `Satisfy` to `SelfReferencingCollectionAssertions` - [#1500](https://github.com/fluentassertions/fluentassertions/pull/1500).
```

Comment:
🤔 I noticed the previous example is wrong too, but those release notes should be as functional as possible. So instead of referring to `SelfReferencingCollectionAssertions`, I would you use a phrase like:

> Added `Satisfy` to be able to compare a collection with a set of predicates in any order.